Urban-Fun/View Controllers/TimelineViewController.m

```diff
@@ -118,18 +120,25 @@ - (void)didTapUsername:(nonnull User *)user {
     self.profileToView = user;
 }
 
+- (void)didTapDistance:(Activity *)activity{
+    self.tappedActivity = activity;
```

Comment:
Instead of creating a new variable for the class, I'd suggest to use the `sender` object for the `performSegueWithIdentifier` function, I don't think you really need this `tappedActivity` object for another reason, so you can pass it down from the tap action
`[self performSegueWithIdentifier:@"googleMaps" sender: activity];`
and then get the object in the `prepareForSegue`
`vc.activity = (Activity *)sender;`

In order to avoid crash, you can also validate if the object is type of activity before setting it:
`[sender isKindOfClass:[Activity class]]`